### PR TITLE
Use parent

### PR DESCRIPTION
**Pull request by aharonha**

### CODE REVIEW

**Review by segabriel — DISMISSED** [review]

File: benchmarks/src/main/java/io/scalecube/services/benchmarks/codecs/ServiceMessageDecodeAndDecodeDataBenchmarksRunner.java [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-import io.scalecube.benchmarks.BenchmarksSettings;
-import io.scalecube.services.api.ServiceMessage;
-import io.scalecube.services.codec.ServiceMessageCodec;
-
 import com.codahale.metrics.Timer;
-
 import io.netty.buffer.ByteBuf;
 import io.rsocket.Payload;
-
+import io.scalecube.benchmarks.BenchmarksSettings;
```
Comment (ronenhamias) [inline benchmarks/src/main/java/io/scalecube/services/benchmarks/codecs/ServiceMessageDecodeAndDecodeDataBenchmarksRunner.java]: the check style seem to be wrong here

**Review by ronenhamias — DISMISSED** [review]
the check style seem to be wrong in import order

**Comment by ronenhamias** [comment]
please update the relevant import order files

**Comment by ronenhamias** [comment]
this issue should wait with merge until import order guidelines are fully cleared.

File: services-api/pom.xml [hunk 2]
```diff
@@ -48,5 +49,4 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
-
-</project>
+</project>
```
Comment (segabriel) [inline services-api/pom.xml]: add new empty line

**Comment by artem-v** [comment]
> what is the status for this PR?

I work on it

**Pull request merged**